about/service-objectives.md

```diff
@@ -0,0 +1,75 @@
+# Service Level Objectives and Principles
+
+This page describes the **Service Level Objectives** of 2i2c's infrastructure services[^slos].
+These describe our goals in running infrastructure for the communities that we serve.
+They indicate what our users can expect when using the infrastructure we support.
+
+We design our infrastructure, and consistently hone our practices, to meet these objectives.
+They evolve over time as we get feedback from communities we serve, and learn more about how to best deliver impact via our services.
+
+:::{note}
+2i2c does not currently have a **Service Level Agreement** (SLAs), as this is generally a legally-binding document that involves calculation of risk via revenue lost during service outages.
+We currently do not have the capacity to design and litigate strict SLAs, and believe that we will have the most impact by instead committing to service **objectives** that are transparent and follow best practices.[^zenodo]
+
+We may revisit this in the future depending on the feedback we get from other communities!
+:::
+
+## High availability
+
+The infrastructure that 2i2c runs should be available to its communities 24/7, and with minimal human intervention needed to maintain this level of performance.
+We invest in continous development to improve the resiliency and efficiency of the infrastructure that we run, following best-practices in service design and engineering in the cloud.
+
+:::{admonition} To be refined...
+It is a known anti-pattern to define an ambiguous SLO like "24/7".
```

Comment:
This is a really good point I was going to raise, nice to see it already written.